pkg/volume/csimigration/plugin_manager_test.go

```diff
@@ -195,12 +130,6 @@ func TestMigrationFeatureFlagStatus(t *testing.T) {
 	for _, test := range testCases {
 		pm := NewPluginManager(csiTranslator, utilfeature.DefaultFeatureGate)
 		t.Run(fmt.Sprintf("Testing %v", test.name), func(t *testing.T) {
-			// CSIMigrationGCE is locked to on, so it cannot be enabled or disabled. There are a couple
-			// of test cases that check correct behavior when CSIMigrationGCE is enabled, but there are
-			// no longer any tests cases for CSIMigrationGCE being disabled as that is not possible.
-			if test.pluginFeature != features.CSIMigrationGCE {
```

Comment:
changing this to `if len(test.pluginFeature) > 0` would let us blank out `pluginFeature` for testcases for permanent on / locked feature gates and still test the migration behavior

Reply:
Thanks for the point! Done.